examples/distro/custom/src/main/java/io/opentelemetry/demo/DemoIdGenerator.java

```diff
@@ -0,0 +1,25 @@
+package io.opentelemetry.demo;
```

Comment:
what do u think of putting everything under `com.example.javaagent` instead of `io.opentelemetry.demo`?